### DISCUSSION TITLE
How do I download PP to my Windows 10 desktop?

### DISCUSSION THREAD

**Discussion by fjp11907** [photoprism/photoprism#4660] [Q&A]
How do I download PP to my Windows 10 desktop?

**Comment by omerdduran** [comment]
Hi. For instructions on how to install PhotoPrism on Windows 10, please see our official Windows installation guide [here](https://docs.photoprism.app/getting-started/docker-compose/#__tabbed_1_5). This guide will walk you through the complete installation process step-by-step.
